go/vt/sqlparser/ast.go

```diff
@@ -1010,6 +1019,7 @@ const (
 	NotRegexpStr         = "not regexp"
 	JSONExtractOp        = "->"
 	JSONUnquoteExtractOp = "->>"
+	CollateStr           = "collate"
```

Comment:
I think this was removed in master and its existence here is a result of merge conflict